Paco Balancing

SirNightKnight opened the pull request:
## About The Pull Request
### Reverts: https://github.com/Monkestation/Monkestation2.0/pull/4663

made paco mags small again, in exchange i made paco lethals harder to get by locking lethal paco ammo behind a higher tier of sec research and behind the armoury in cargo. increased paco and wt rubbershot stamina damage from 35 to 50. wt saltshot got buffed from 35 to 40. Added another Paco to the Paco weapon crate to match the ammo crate changes

## Why It's Good For The Game
if the issue the reverted PR was trying to fix was sec being overloaded with paco lethals why not just make said lethals harder to get. my reasoning for the stamina damage increases so paco's are more of a sidegrade to disablers. wt ammo stamina buffs are so they are on par with disabler beams. 

quick comparison: 

Disabler
- 20 shots total
- no recoil
- 5 shots to stam crit
- can shoot through windows
- rechargeable 
- applies debilitated stacks  which causes the target to drain more stamina whenever they take additional stamina damage

Paco with  Rubbershot
- 16 shots total
- recoil
- 5 shots to stam crit
- can't shoot through windows
- non-rechargeable/cannot be EMPed
- might get sued by lawyers for bruises caused by using rubbershot

im pretty sure no other magazine is normal sized either.

## Changelog

:cl:
balance: Paco mags are small again.
balance: locked lethal Paco ammo behind the "Advanced Weaponry" tech node.
balance: split Paco ammo between non-lethal ammo as a security order and lethal ammo as an armoury locked order. added another Paco to the Paco weapon crate.
/:cl:

Wisemonster commented:
Don't ammo benches have the ability to print lethals easily? Ammo benches are both orderable from cargo, and are in security on some maps, so this is just gonna make sec order an ammo bench round-start (unless you nerf it in some way), or just load up the one they already have to mass print lethals. Not to mention, the pr being reverted doesn't specify lethals, it just talks about having a shit ton of paco mags.

Gw0sty commented:
> Don't ammo benches have the ability to print lethals easily? Ammo benches are both orderable from cargo, and are in security on some maps, so this is just gonna make sec order an ammo bench round-start (unless you nerf it in some way), or just load up the one they already have to mass print lethals. Not to mention, the pr being reverted doesn't specify lethals, it just talks about having a shit ton of paco mags.
> 
> Edit: Bullet drives also exist, which is another way for sec to get lethals outside of tech.
<details>
<summary>Ammo Benches.</summary>
Ammo benches required to be unlocked with a special disk for the ability to print lethal ammo. Otherwise its mainly rubber and standard ammo types. Critically thing is here Ammo benches cant print paco ammo, at all. 35 Lethal nor nonlethal can be printed. 

While I do believe that Ammo benches and how cheap/accessiable import mags/bullets boxes(50 credits each) are in imports is an issue, that's still an import issue and is unrelated to the PR.
</details>
<details>
<summary>Bullet drives. </summary>
Bullet drives don't print lethal ammo, nor any type of ammo at all. They are actually just reskinned dish drives for bullets. The things that pick up dishes for people when they are done in the kitchen, but this one moves the bullets to the trash after. Its a glorified cleanbot frankly.

</details>

For the PR itself. Halfing the the amount of mags you obtain from a crate is good. I would probably recommend upping the cost from armory ammo 2 to 3 or 4 so its not a straight 50/50 between lethal and nonlethal.

As for adjusting the lethal paco ammo research I'm pretty iffy. Specifically not because its moving it to a later research but an out-of-the-way research. Basic weapon tech is needed for advanced cybernetics and mechs. Which incentivises the roboticists to research them by proxy. Advanced weapon tech doesn't have anything science typically goes after besides one point grav guns. Meaning it has to be researched for its own sake. I'm no expert on gun balance but that's important to note for the research side of things.

Wisemonster commented:
> Ammo Benches.
> Ammo benches required to be unlocked with a special disk for the ability to print lethal ammo. Otherwise its mainly rubber and standard ammo types. Critically thing is here Ammo benches cant print paco ammo, at all. 35 Lethal nor nonlethal can be printed.
> While I do believe that Ammo benches and how cheap/accessiable import mags/bullets boxes(50 credits each) are in imports is an issue, that's still an import issue and is unrelated to the PR.

You mean the disk that is right next to the ammo bench board in the cargo list? Also, where did you get the info they can't print any paco ammo? Cause I built an auto lathe using both the board and disk ordered from cargo, and was able to print both ammo fine.

![Rashidun Waypoint XCII 3_2_2025 11_55_04 PM](https://github.com/user-attachments/assets/ff558558-23f0-43ca-b02a-0c80f5eea8f6)

Gw0sty commented:
I got the info from my own testing. 
When I did it I got nothing for ammo subtypes.
This is strange...
![Screenshot 2025-03-02 201744](https://github.com/user-attachments/assets/48682a2c-e806-4500-9a55-c0a7f09da333)

Wisemonster commented:
> I got the info from my own testing. When I did it I got nothing for ammo subtypes. The disk was uploaded and everything This is strange... ![Screenshot 2025-03-02 201744](https://private-user-images.githubusercontent.com/83688318/418440320-48682a2c-e806-4500-9a55-c0a7f09da333.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA5ODExNTksIm5iZiI6MTc0MDk4MDg1OSwicGF0aCI6Ii84MzY4ODMxOC80MTg0NDAzMjAtNDg2ODJhMmMtZTgwNi00NTAwLTlhNTUtYzBhN2YwOWRhMzMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzAzVDA1NDczOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4YTRlZTBkZmRkOWFmYzdiYzcxM2E2MzRkMThhM2ZhYWYwYWQzMDBlYzRlZWY0YWIzN2NlNDRiMDU2MWUyNTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ppMeTZhY5aIzGYRb2cOYT6BwoE77TeuTd-h00oMtXqw)

Did you load the disk before sticking a mag in? Cause loading a disk while a mag is in won't update the ammo until you take the mag out and put it back in.

Gw0sty commented:
Ah, yeah that was, it. Folly on my part then.

SirNightKnight commented:
> > Don't ammo benches have the ability to print lethals easily? Ammo benches are both orderable from cargo, and are in security on some maps, so this is just gonna make sec order an ammo bench round-start (unless you nerf it in some way), or just load up the one they already have to mass print lethals. Not to mention, the pr being reverted doesn't specify lethals, it just talks about having a shit ton of paco mags.
> > Edit: Bullet drives also exist, which is another way for sec to get lethals outside of tech.
> 
> Ammo Benches.
> Bullet drives.
> For the PR itself. Halfing the the amount of mags you obtain from a crate is good. I would probably recommend upping the cost from armory ammo 2 to 3 or 4 so its not a straight 50/50 between lethal and nonlethal.
> 
> As for adjusting the lethal paco ammo research I'm pretty iffy. Specifically not because its moving it to a later research but an out-of-the-way research. Basic weapon tech is needed for advanced cybernetics and mechs. Which incentivises the roboticists to research them by proxy. Advanced weapon tech doesn't have anything science typically goes after besides one point grav guns. Meaning it has to be researched for its own sake. I'm no expert on gun balance but that's important to note for the research side of things.

i increased the lethal cost to 800 (so 4 times the cargo crate price) but also added another mag and ammo packet. i moved it to advanced weaponry since that's also when lethal WT ammo is unlocked.

RikuTheKiller commented:
>non-rechargeable/cannot be EMPed

Being rechargeable is objectively the worst part about disablers. First off, it means there's no way to get more ammo while in combat. That's extremely important. The 4 point difference in mag size doesn't matter when you have another 100 shots worth of rubbershot in your bag. That is the reason they were made normal-sized, not lethals.

Second, you might think recharging is going to be a good thing when you finally run out of all the shots you brought. Well, no. Because you can print an ammo box full of rubbershot, take out the mag, click the mag with the ammo box, put it back in and then push back the slider. This takes a few seconds once you're used to it compared to the 20s or so of a disabler recharge.

Overall, using magazines instead of recharging is an extremely big upside to ballistics, if not literally the best part about them.

SirNightKnight commented:
> > non-rechargeable/cannot be EMPed
> 
> Being rechargeable is objectively the worst part about disablers. First off, it means there's no way to get more ammo while in combat. That's extremely important. The 4 point difference in mag size doesn't matter when you have another 100 shots worth of rubbershot in your bag. That is the reason they were made normal-sized, not lethals.
> 
> Second, you might think recharging is going to be a good thing when you finally run out of all the shots you brought. Well, no. Because you can print an ammo box full of rubbershot, take out the mag, click the mag with the ammo box, put it back in and then push back the slider. This takes a few seconds once you're used to it compared to the 20s or so of a disabler recharge.
> 
> And if that's not good enough, then keep in mind that one ammo box (50) can fill 3 mags (48) worth of rubbershot. So that few seconds scales far better than disablers unless you build extra rechargers so you can charge multiple in parallel.
> 
> Overall, using magazines instead of recharging is an extremely big upside to ballistics, if not literally the best part about them.
> 
> > recoil
> 
> Also recoil is a worthless downside because borbop added gun wielding, just press shift+x and your problems disappear. Pretty sure Paco has so little recoil it gets reduced all the way to 0 when wielding it, so yeah.

unless it got changed in the last month all guns have the same exact recoil amount. recoil on the paco shifts your screen about 2-3 tiles every shot and wielding it to mitigate recoil means you gotta unwield it to reload which can be obtuse in the heat of combat.

 honestly the rest of your points are fair. i assumed they meant lethals cus they were using the term john shitsec and i have a vague memory of them talking about sec loading up on paco lethals being a problem. 

another thing is you gotta sacrifice inventory space for magazines.

MilkForever commented:
Ammo boxes would be a lot better if filling magazines were a one-by-one do_after instead of instantly filling the mag with no delay.
